src/OpenTelemetry.PersistentStorage.FileSystem/README.md

```diff
@@ -14,40 +14,62 @@ exporters to improve the reliability of data delivery.
 dotnet add package OpenTelemetry.PersistentStorage.FileSystem
 ```
 
-## Basic Usage
+## Usage
 
 ### Setup FileBlobProvider
 
 ```csharp
-using var persistentBlobProvider = new FileBlobProvider("test");
+using var persistentBlobProvider = new FileBlobProvider(@"C:\temp");
 ```
 
-Following is the complete list of configurable options that can be used to set
-up FileBlobProvider:
+#### Configuration
 
-* `path`: Sets folder location where blobs are stored.
+Following is the complete list of parameters that `FileBlobProvider` constructor
+accepts to control its configuration:
 
-* `maxSizeInBytes`: Maximum allowed folder size. Default is 50 MB.
+##### path
 
-* `maintenancePeriodInMilliseconds`: Maintenance event runs at specified interval.
-Default is 2 minutes. Maintenance event performs the following tasks:
+Sets directory location where blobs are stored. `Required`.
 
-  * Removes `*.blob` files for which the retention period has expired.
-  * Removes `*.tmp` files for which the write timeout period has expired.
-  * Updates `*.lock` files to `*.blob` for which the lease period has expired.
-  * Updates `directorySize`.
+##### maxSizeInBytes
 
-* `retentionPeriodInMilliseconds`: Retention period in milliseconds for the blob.
-Default is 2 days.
+Maximum allowed folder size. `Optional`. Default if not specified: `52428800`
+bytes.
 
-* `writeTimeoutInMilliseconds`: Controls the timeout when writing a buffer to
-blob. Default is 1 minute.
+New blobs are dropped after the folder size reaches maximum limit.
```

Comment:
I think we should do something like...

```
New blobs are dropped after the folder size reaches maximum limit. A log message is written if blobs cannot be written. See [Troubleshooting](#troubleshooting) for more information.
```

Then add a section a the bottom for "Troubleshooting" which mentions the event source and links to self-diagnostics feature for how to capture that info?

Reply:
added